app/client.py

```diff
@@ -29,6 +42,11 @@ def get_spocs(self):
         except Exception as e:
             logging.warning("Could not target based on geolocation. {0}".format(str(e)))
 
+        if self.country:
```

Comment:
When we pass in explicit overrides, should we worry about verifying the `geolocation_provider` object?

Reply:
No, but the intended use of these parameters is development and QA, so it won't impact production.